<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>671</width>
      <height>481</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>510</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>ReadOnly</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="langBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>510</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+Q</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="openEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>580</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>580</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="quitEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>580</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="keylabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Keyboard shortcut:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="themeBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>580</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="themelabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>550</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Theme:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
